data/json/items/tool/lighting.json

```diff
@@ -509,6 +509,52 @@
     },
     "flags": [ "LIGHT_10", "TRADER_AVOID", "FIRE", "ALLOWS_REMOTE_USE", "WATER_EXTINGUISH" ]
   },
+  {
+    "id": "oil_lamp_clay",
+    "type": "TOOL",
+    "name": { "str": "clay oil lamp (off)", "str_pl": "clay oil lamps (off)" },
+    "description": "This is a specialized small clay container with a wick, an ancient form of oil lamp.  Unlike newer designs, it isn't windproof, so the flame is likely to be extinguished if exposed to wind; better use it indoors.  It does not provide much light, but it lasts a long time.  Use it to light it.",
+    "weight": "620 g",
+    "volume": "300 ml",
+    "price": 800,
+    "price_postapoc": 100,
+    "bashing": 1,
+    "material": [ "clay" ],
+    "symbol": ";",
+    "color": "brown",
+    "ammo": [ "lamp_oil" ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "watertight": true, "ammo_restriction": { "lamp_oil": 250 } } ],
```

Comment:
Is it really watertight? If you take an oil lamp with charges into e.g a river, the oil should be lost because theres multiple holes in the lamp.

Reply:
"watertight" appears to define whether it can hold a liquid at all or not; it's "open_container" that affects if it would not contain the liquid underwater or in a backpack. Which turns out to not work as intended for magazines; you can't place it into a backpack while it holds a liquid, but once you do put it in, you can load lamp oil in it, which won't spill. I've mentioned that issue above.